internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -263,6 +263,44 @@ func (vc *WSVersionCompatibility) processConfigTableUpdates(uncompressedPayload
 	return processedPayload, nil
 }
 
+func (vc *WSVersionCompatibility) removeCoreEntity(
+	processedPayload string,
+	entityType string,
+	dataPlaneVersionStr string,
+	configTableKey string,
+	changeID ChangeID,
+	tracker *ChangeTracker,
+) string {
+	entity := gjson.Get(processedPayload, fmt.Sprintf("config_table.%s", configTableKey))
+	for _, res := range entity.Array() {
+		var err error
+		id := res.Get("id").String()
+		err = tracker.TrackForResource(changeID, ResourceInfo{
```

Comment:
nit: Can get rid of `var err error` and just do `err := tracker...` instead

Reply:
fixed this.